clippy_lints/src/unwrap.rs

```diff
@@ -32,6 +32,28 @@ declare_clippy_lint! {
     "checks for calls of unwrap[_err]() that cannot fail"
 }
 
+/// **What it does:** Checks for calls of unwrap[_err]() that will always fail.
```

Comment:
Rendered in the docs this will look like:

> **What it does:** Checks for calls of unwrap[_err]() that will always fail.

So you should probably put the `unwrap[_err]()` in back ticks to mark it as code :)

Reply:
Thanks! It's nicer with back ticks anyway.